pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -40,7 +40,7 @@
 , static ? stdenv.hostPlatform.isStatic
 , enableOptimizations ? false
 # enableNoSemanticInterposition is a subset of the enableOptimizations flag that doesn't harm reproducibility.
-, enableNoSemanticInterposition ? true
+, enableNoSemanticInterposition ? !stdenv.cc.isClang
```

Comment:
In any case we should leave a comment why it doesn't work on there.

Reply:
What it says right now is: not on clang. I left more details on the commit message, isn't that sufficient?

Reply:
commit messages aren't as discoverable when looking at the expression.

Reply:
Fair enough.